crates/datagen/src/lib.rs

```diff
@@ -1825,3 +2024,17 @@ impl<'a> RecordGenerator<'a> {
         Ok(self.json_obj.as_ref().unwrap())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{decimal_max, decimal_min};
+    #[test]
+    fn rust_decimal() {
```

Comment:
what is this testing?

Reply:
that we don't panic when generating all p,s combos